core/roboconf-target-iaas-openstack/src/main/java/net/roboconf/target/openstack/internal/OpenstackIaasHandler.java

```diff
@@ -90,12 +91,17 @@ public String createMachine(
 			String messagingIp,
 			String messagingUsername,
 			String messagingPassword,
-			String rootInstanceName,
+			String scopedInstancePath,
 			String applicationName )
 	throws TargetException {
 
 		this.logger.fine( "Creating a new machine." );
 
+		// For IaaS, we only expect root instance names to be passed
+		if( InstanceHelpers.countInstances( scopedInstancePath ) > 1 )
+			throw new TargetException( "Only root instances can be passed in arguments." );
```

Comment:
IMO, this is purely theoretical.
In practice, I am not sure it would work.

The main goal of this upgrade is for Docker, agents in-memory - for tests - and embedded targets, even if I am not sure for this last one. Until someone complains about this limitation, I think it would be better to keep it this way. Besides, this is an easy opportunity for users to contribute some code (convert an exception into a log entry). ^^